### ISSUE TITLE
fix graphml labels

### ISSUE THREAD

**Issue by andrewsu** [biothings/biothings_explorer#52]
When exporting to graphml via `to_graphml`, it looks like node labels aren't being handled correctly:

```
    <node id="CACNA1C">
      <data key="d1">bts:symbol</data>
      <data key="d0">Gene</data>
      <data key="d2">2</data>
      <data key="d14">[,',E,N,S,G,0,0,0,0,0,1,5,1,0,6,7,',,, ,',E,N,S,G,0,0,0,0,0,2,8,5,4,7,9,',]</data>
      <data key="d15">[,',1,3,9,0,',]</data>
      <data key="d16">[,',1,1,4,2,0,5,',]</data>
      <data key="d17">[,',7,7,5,',]</data>
      <data key="d18">[,]</data>
      <data key="d12">[,',C,1,4,1,3,0,5,7,',]</data>
      <data key="d19">[,',H,s,.,6,9,0,0,1,0,',,, ,',H,s,.,1,1,8,2,6,2,',]</data>
      <data key="d20">[,',P,A,8,3,',]</data>
      <data key="d10">[,',v,o,l,t,a,g,e,-,g,a,t,e,d, ,L,-,t,y,p,e, ,c,a,l,c,i,u,m, ,c,h,a,n,n,e,l, ,C,a,v,1,.,2, ,a,l,p,h,a, ,1, ,s,u,b,u,n,i,t,,, ,s,p,l,i,c,e, ,v,a,r,i,a,n,t, ,1,0,*,',,, ,',c,a,l,c,i,u,m, ,c,h,a,n,n,e,l,,, ,L, ,t,y,p,e,,, ,a,l,p,h,a,-,1, ,p,o,l,y,p,e,p,t,i,d,e,,, ,i,s,o,f,o,r,m, ,1,,, ,c,a,r,d,i,a,c, ,m,u,s,c,l,e,',,, ,',c,a,l,c,i,u,m, ,v,o,l,t,a,g,e,-,g,a,t,e,d, ,c,h,a,n,n,e,l, ,s,u,b,u,n,i,t, ,a,l,p,h,a,1, ,C,',,, ,',c,a,l,c,i,u,m, ,c,h,a,n,n,e,l,,, ,c,a,r,d,i,c, ,d,i,h,y,d,r,o,p,y,r,i,d,i,n,e,-,s,e,n,s,i,t,i,v,e,,, ,a,l,p,h,a,-,1, ,s,u,b,u,n,i,t,',,, ,',v,o,l,t,a,g,e,-,d,e,p,e,n,d,e,n,t, ,L,-,t,y,p,e, ,c,a,l,c,i,u,m, ,c,h,a,n,n,e,l, ,s,u,b,u,n,i,t, ,a,l,p,h,a,-,1,C,',,, ,',D,H,P,R,,, ,a,l,p,h,a,-,1, ,s,u,b,u,n,i,t,',,, ,',c,a,l,c,i,u,m, ,c,h,a,n,n,e,l,,, ,v,o,l,t,a,g,e,-,d,e,p,e,n,d,e,n,t,,, ,L, ,t,y,p,e,,, ,a,l,p,h,a, ,1,C, ,s,u,b,u,n,i,t,',,, ,',v,o,l,t,a,g,e,-,d,e,p,e,n,d,e,n,t, ,L,-,t,y,p,e, ,C,a,2,+, ,c,h,a,n,n,e,l, ,a,l,p,h,a, ,1, ,s,u,b,u,n,i,t,',]</data>
      <data key="d21">[,',C,A,C,N,A,1,C,',]</data>
    </node>
```

A full graphml output file is here: [sample_graphml.txt](https://github.com/biothings/biothings_explorer/files/4257148/sample_graphml.txt)

**Comment by kevinxin90** [comment]
Fixed in 92dc678504ee44f0ab0abb1b5f88e3dee55036ea

**Comment by kevinxin90** [comment]
sample output: 
[sample_graphml.txt](https://github.com/biothings/biothings_explorer/files/4308943/sample_graphml.txt)
